docs/setup.md

```diff
@@ -0,0 +1,35 @@
+In order to use the react eslint plugin do the following...
+
+1. At the root of your project type <code>npm install eslint-plugin-react --save-dev</code>
+2. Check your package.json 'devDependencies' for the following entries (version numbers may vary)...
+<pre>
+  "devDependencies": {
+    "eslint": "^3.19.0",
+    "eslint-plugin-react": "^6.10.3"
+  }
+</pre>
+3. In your eslint configuration file (.eslint.rc / package.json etc.) add the following under 'esLintConfig'...
```

Comment:
`esLintConfig → eslintConfig`, but also that key only applies to package.json. In the far more common `.eslintrc` (not `.eslint.rc`) you just put the json object directly.

Reply:
...and this stuff really makes me cry... :o)